nuclear_simulator/simulator/core/sim.py

```
@@ -0,0 +1,945 @@
+            try:
+                maintenance_work_orders = self.maintenance_system.update(elapsed_minutes, self.dt)
+                # Add maintenance info to result if work orders were created/executed
+                if maintenance_work_orders:
+                    info['maintenance_work_orders'] = [wo.to_dict() for wo in maintenance_work_orders]
+            except Exception as e:
+                warnings.warn(f"Maintenance system update failed: {e}")
+        
+        # Collect states using the new state management system
+        if self.enable_state_management and self.state_manager is not None:
+            try:
+                collected_states = self.state_manager.collect_states(current_datetime)
```

Ruff (0.11.9)

Local variable `collected_states` is assigned to but never used

Remove assignment to unused variable `collected_states`

(F841)